src/locale/languages.ts

```diff
@@ -35,6 +36,7 @@ export const APP_LANGUAGES: AppLanguageConfig[] = [
   {code2: AppLanguage.pt_BR, name: 'Português (BR) – Portuguese (BR)'},
   {code2: AppLanguage.uk, name: 'Українська – Ukrainian'},
   {code2: AppLanguage.ca, name: 'Català – Catalan'},
+  {code2: AppLanguage.ja, name: '简体中文（中国） – Chinese (Simplified)'},
```

Comment:
I think it should be AppLanguage.zh_CN here

Reply:
Thanks for the review, I've fixed the issue and cross-checked the translation with my friend.